admin/partials/boldgrid-backup-admin-test.php

```diff
@@ -450,6 +450,19 @@
 	echo $fail_tips; // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
 
 	echo $table; // phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped
+
```

Comment:
The coding to calculate the number of notices that would be added based on test results was left in there, however the code to actually display it was taken out, since we didn't really want that feature yet. If you want the code used to calculate that taken out as well, that is not a problem